images/master/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM google/debian:wheezy
```

Comment:
Do we really need a full debian base?  How far are we from being able to run FROM SCRATCH?  Do we even know what things we would need?